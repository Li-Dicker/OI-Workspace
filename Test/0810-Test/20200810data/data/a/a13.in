10
198
82219 17658 37033 77282 20201 97433 67994 89093 43141 33081 95686 84943 79717 61232 67250 66724 90123 27022 57234 7056 1401 41054 56062 49773 85174 39218 87375 82859 11273 44482 99643 42043 59712 74432 151 71008 48013 88577 18664 90249 56469 82201 37759 93202 98114 19166 53863 76312 73114 45777 62790 60929 63849 47700 71817 64352 89004 73677 31280 14012 5201 14153 75785 40926 7102 85224 72446 38206 6130 94490 1367 76223 32056 28140 51588 86881 85410 80426 27360 64328 91877 36385 38049 90185 88914 33876 36953 68488 91688 51109 17715 98642 12266 7937 33621 82362 9378 6193 60998 75464 50259 70243 38506 70225 65245 16462 26919 81360 16105 13816 27242 8901 98372 30435 93235 35734 74946 62851 42530 55741 31210 23574 10159 88015 47933 44041 50238 80401 85493 77110 89286 92364 14657 44399 58726 11837 67770 38496 43849 77309 82775 6214 43848 2889 24807 43105 33989 48128 546 7526 32145 72210 34243 57831 98583 25462 14649 78014 58635 37761 84284 21363 61031 18262 89481 39261 82735 20890 41712 39803 93804 88410 28829 97038 80532 55509 90392 2606 37767 63235 91774 91647 31324 50980 945 40404 84308 1775 27881 96975 4810 73553 95317 75344 3069 95512 41770 83391 
32508 36494 38111 44498 5730 47255 3116 6708 24196 45810 11671 33029 11204 46436 4304 38065 42325 15060 14716 2232 20892 27536 15943 30711 28489 12086 24949 10408 29037 25608 28529 46526 44053 166 20879 30940 25813 13525 42344 34226 3927 30108 36276 36120 37438 13481 21911 27258 23138 3669 1596 37494 38787 5121 42570 19740 7167 6022 34605 13445 43171 1153 43173 49480 38845 24314 49079 46750 20858 45622 44435 4603 6267 9449 9215 12556 44000 4190 20288 39671 3858 32242 27627 26985 15864 7325 47850 8888 35123 20355 43204 45020 39463 3323 11266 41197 26402 29993 19110 2314 4157 49746 28217 24092 24865 13855 3089 31688 16981 14799 16507 15684 26049 47694 24370 2417 48881 44165 18869 47502 26997 36133 28857 34809 26749 15215 46092 6850 16215 36867 21166 36277 37277 42845 45587 41316 3009 45421 42219 25587 31730 19801 19730 45655 35782 28318 43765 42132 8030 34982 18941 30086 28528 43399 19699 7651 1423 18253 19928 23379 6752 44273 26812 7601 31808 46657 35719 20459 26416 4515 35086 18420 38452 23365 10866 16846 9084 28635 42537 43375 4184 2211 44452 45269 19891 2850 9375 16689 36437 36235 41242 30440 34089 9126 29358 40817 36901 43944 
200
11023 13610 982 9974 71131 8056 18467 1884 79621 68895 69139 35724 69377 67017 64488 2740 1762 96458 72150 60583 55115 4004 92207 48612 77098 4388 96408 6329 45210 32305 69780 65743 94571 70872 32000 58757 31220 25308 54455 30267 89040 16726 50281 68891 87391 24461 11402 55976 63449 29261 94973 68388 75900 36128 42590 13065 5923 17211 71549 91722 41369 75680 67783 83119 15335 32909 28208 57153 34686 38374 48377 51848 87309 23844 15881 98997 90605 62680 7894 85003 65787 58215 10369 12444 56675 78498 80158 95125 63531 68049 26116 42988 48148 68848 74370 56963 87644 84886 76184 13625 58379 12997 92222 21222 23923 84569 25509 67992 63361 89343 74378 51893 27918 7747 9720 69890 89245 79042 31104 62642 35334 56022 24500 11955 85793 98285 92125 85252 33792 97239 11628 47955 67833 80959 89292 14453 61422 70057 81158 37597 35769 2500 25013 31734 50932 96016 795 72077 42278 36514 11988 29264 98795 62180 11334 65162 99350 63464 64355 90792 95817 91260 17742 98010 36553 40769 26515 18905 13171 41844 44078 44705 95533 42589 81895 19381 86756 80215 16175 1200 69754 28235 83487 25018 39341 57936 90534 87203 43515 85327 65935 40455 71666 37401 838 18017 31756 83006 10574 45090 
22587 2989 15021 35352 31624 21700 19518 22725 32069 34759 24894 9318 14237 30248 15385 8486 29696 1310 18291 10266 3199 29108 7329 30995 24305 36859 11819 31677 35450 12779 10293 9329 11612 27563 49735 31302 19671 85 27131 25849 19111 42493 31791 16108 3193 20044 13478 47251 41571 22340 48838 47363 15414 42558 46398 7324 33047 46218 35501 2059 20258 46674 36409 12805 10754 37276 23012 37023 30281 45463 11292 9940 39521 3298 5729 46181 22912 43192 19196 48266 29660 20549 14543 14707 18675 10194 21738 38206 1933 343 39731 3439 21226 11815 2970 6589 37731 23451 35335 16576 34246 3627 3509 9063 38887 5020 38896 6722 20081 31006 30918 35937 46843 17610 40049 14640 29659 19208 46905 3588 22764 12944 42555 34400 40555 30918 18683 14085 27468 8786 21618 18183 3756 4863 43194 24541 11126 18299 42429 410 26158 49297 18764 18828 33559 26196 9550 28849 21581 8510 49590 41654 27734 27403 2694 26595 22814 1834 12298 18298 19732 1838 26108 1089 40955 35664 41504 18392 40829 37214 28865 25130 12995 9030 18695 33761 620 21097 24223 44138 4883 9059 13692 35469 19364 38498 38195 36490 32953 48978 122 45750 49451 14256 7188 35565 3914 15264 39550 20169 
198
79893 30651 39347 74076 38911 1031 60522 62919 1322 52489 61147 90087 64396 21785 57373 62802 54741 37826 76091 41179 31977 4811 77753 13082 64258 83962 31552 40787 642 10851 10285 70012 54785 59336 21676 55380 679 63696 22290 81035 83846 30269 34373 16942 25241 93828 41952 26731 52442 71829 79735 92660 67197 34077 56650 26597 79716 34065 903 47762 2990 38891 58960 65770 54059 16726 4182 23643 20372 73253 95349 71455 2029 73351 7699 9142 15724 26920 73621 92646 25920 13689 97561 3470 53589 11730 93074 56287 2094 37171 79909 99320 49801 74167 88074 79800 97616 48835 40423 9633 44498 53180 25428 40729 55028 82687 55173 76726 25490 33927 13541 80825 94821 15923 24105 9197 9725 2209 75416 11836 74906 4109 81899 17072 47352 98232 42639 78407 75456 50822 60841 55435 24156 8879 74650 41187 63466 41139 31088 85644 63958 4594 45992 38062 29645 88624 3051 92890 31034 19113 36151 57149 23576 62087 74408 40687 46011 51912 13447 45967 69639 70382 86228 86758 36941 63828 85384 75997 57586 51344 38884 87138 94747 53482 4424 50174 15575 46477 58325 24595 36089 99174 96439 36359 24387 20235 31241 90095 45315 20124 2623 47104 42472 71587 83155 80150 68418 92341 
4141 42472 27670 44045 32892 46792 37157 48619 14455 16364 27769 28399 46612 10686 11238 7535 15961 25437 148 28253 3463 22893 6376 17858 38322 43646 15806 11825 13798 10844 15417 43220 27355 28820 26680 13686 2360 25892 16336 38393 16402 294 891 17437 11050 41907 39442 21035 8727 44740 21298 39589 34165 43535 14986 30619 11672 6544 40705 26035 26245 44875 26252 10844 26669 49828 33581 40741 13859 17011 10606 40396 25737 23658 21903 20147 26588 11128 48095 29744 41218 17928 17244 10160 45012 37842 11128 40524 34669 527 12148 4696 11354 5793 45474 31020 5529 35680 23916 1933 35354 4107 677 41358 23522 22538 25312 25335 7734 22325 22386 21427 20693 4167 11111 20504 42250 15635 28238 35004 14236 1655 26342 15595 18801 1388 12086 17373 7778 38636 46671 5082 45832 44629 22101 32522 31496 1923 47022 40434 9677 24678 45784 21455 30974 20526 10511 32535 40029 22642 13280 1509 36923 4615 32630 34093 4467 469 46415 5272 7213 29579 7039 26912 47723 11536 40051 42686 18302 7198 10819 44950 43811 46995 23825 33127 49669 21355 26152 39206 41476 37336 21410 16422 30005 38536 5499 24856 30446 37354 40581 49043 32141 38720 21239 3473 27618 26395 
197
77235 23850 98939 46147 65004 23069 99053 8279 35702 52725 14059 82516 74778 17892 27283 76061 91718 27020 11659 5746 90887 12825 63590 72320 84287 82578 87858 82402 82414 48884 61608 80594 44580 81437 82901 77976 20854 51019 32730 54540 29884 22229 62140 64836 77916 76294 35474 5667 28917 157 58431 53563 75408 68831 49746 20427 99176 63629 1794 4255 9099 54279 50314 52999 10058 54022 20588 26799 60764 53527 77642 39015 1196 48070 63710 80357 94318 98154 1263 21110 66771 90907 76469 26624 81385 509 73058 71085 84433 67854 34592 40221 1506 96891 77111 8708 55085 45871 23474 14914 22403 33543 88033 54951 72982 784 24221 31128 67445 61115 88053 86960 68849 36075 47617 93252 15158 86226 28733 25732 35450 73982 72192 526 9597 86907 75769 54940 48066 69221 2550 87735 28761 77630 72 84271 50508 1748 68881 30514 86545 54155 1647 85864 44516 30051 65018 3675 9596 82910 28356 34967 5201 13268 74006 5052 68641 26541 94738 15303 11604 61103 33142 72599 81361 23186 71386 18537 58798 57660 37591 6590 29528 49859 71000 84076 52301 81935 71006 95127 43330 5070 4978 72520 54613 66142 94890 57286 92452 60432 5614 65946 76278 2666 214 9876 70408 
34279 11562 10700 113 18974 25430 2243 7766 16831 10275 23549 44085 17172 40817 25372 36645 33706 48377 13271 10948 38153 5637 24451 4815 20535 39545 8585 4412 37946 26444 37041 20386 11313 47699 35482 34219 7740 32733 44121 3014 24125 1005 35487 20665 40498 1442 36372 22244 35742 42736 4277 48709 919 382 8 14683 12258 29407 22747 14562 49942 45508 8509 39571 38736 24315 28335 48727 42982 28802 23650 38021 29583 37468 24041 14002 40326 45908 25147 12057 42010 23930 13926 37875 554 30242 34829 41514 46320 23757 18590 4524 30208 19576 10391 44642 23873 36877 25923 35281 8238 18389 33690 26132 16425 38233 37983 4380 15368 34836 16963 23605 22952 42936 21976 5251 32343 3779 39698 32116 2149 38748 23816 20813 18858 9972 35881 44518 23748 49806 13519 27619 32262 6269 21610 49372 13362 6312 35639 7943 5065 49176 11944 19898 31802 49339 6464 38392 36731 42283 34732 41260 30923 9485 14730 39901 20140 26622 48548 18440 43991 17869 40206 32921 36173 29961 27151 22506 27080 977 15781 31509 20548 35436 22937 49528 15534 14466 3383 34452 26237 27781 25901 34408 37187 39704 19033 2068 41399 30847 39886 9792 5738 10303 46069 49039 28374 
200
26142 46799 55516 42616 28531 37056 64629 39910 64367 4229 18857 38279 74250 33964 49088 95567 87350 54250 84486 54812 43216 56973 77857 50495 71940 20625 92635 83884 69462 34262 27978 1343 97741 90579 56858 14048 80041 37332 72818 56726 52916 47555 78218 36548 15976 95986 83749 10205 64306 56787 72073 78793 31293 17567 74491 54539 4877 71057 11659 46292 55919 20471 28819 68885 6271 13836 30504 25049 78818 67644 17197 31360 49409 68199 93772 13407 81477 41273 83268 26475 6408 59960 40156 3457 99120 50252 6758 76979 87580 84411 74214 11172 43339 25854 40353 87431 30333 20564 80546 51107 85983 29211 51595 39909 84004 26772 29592 33748 76707 29347 27020 35903 36597 84375 37152 48636 17705 98892 2235 71023 56719 67335 49202 53309 47555 16446 10627 32357 68278 37883 3099 56394 1694 10225 77331 40397 78903 69189 61083 18361 99650 34440 66192 51436 92491 63668 141 86697 62639 22046 39335 32280 6549 68865 97559 98312 4151 38514 38547 85300 95327 74842 69474 57601 83711 46372 49912 72776 60906 24988 22926 37198 56260 9333 44026 69661 23057 15069 44774 8062 32625 32512 43934 8899 11571 97330 15898 26006 1235 92560 36013 18258 42614 56866 8570 18901 4480 55249 34514 27837 
32943 4435 2718 43205 649 38053 10359 8801 47168 29202 22335 34773 43519 39345 6476 43814 25403 38029 48987 38754 19190 27345 49308 8862 15686 35836 43272 38740 1632 29533 26949 30237 35866 20957 38706 36172 45633 46439 17608 37665 15913 19577 41644 37615 5364 20261 17521 40766 26987 46087 26148 944 4099 48625 26150 15990 1368 34121 14979 42923 10375 21188 16500 29729 25918 24870 27261 30953 1807 21315 7894 1784 13481 4582 15973 29603 16140 42665 18732 45993 24249 30505 20436 22012 43237 35715 35106 22695 25077 30523 11459 26763 35208 49517 5298 47684 30858 38963 12700 3754 42494 45452 38333 6001 48312 19088 31373 33491 23943 43476 21701 14000 8062 17886 332 19559 32667 47248 9147 9680 485 31238 32205 37178 35973 17999 1096 31701 40558 6345 28320 30809 45802 43712 31874 22412 44408 32403 451 32778 28318 2034 5227 10315 45857 41319 20179 11214 34294 29130 28438 37323 21223 24923 48125 10061 41144 47323 35140 30773 10729 37390 33972 44558 24332 19213 10075 1373 31980 46629 12376 25383 7013 22493 15369 4979 6459 44637 21873 41115 15901 39995 25952 36325 37128 16075 22829 29914 22982 9618 38318 9731 12592 7974 28485 18117 42081 1633 23841 7051 
200
95718 27125 81364 5089 97354 57536 97431 89498 75222 92589 41437 40386 82478 33586 14260 41109 4140 21740 51240 88560 61032 47774 60905 85064 37030 24462 92131 62155 55031 75059 93599 18472 71221 51829 30351 86174 90323 46901 92634 54404 21050 75371 87880 59762 46530 87614 8124 28132 56885 50454 9182 74995 39649 50060 93031 12566 74279 67850 43843 39039 42570 98287 37125 38722 59055 3608 1535 92971 19034 29689 79068 45621 42216 75699 67407 99472 25565 54306 4189 78095 8634 97247 99947 33906 17733 61548 56585 61031 3312 5446 41313 99765 50391 35614 57885 98859 3913 32425 47974 55873 52879 39857 74010 23781 44625 45954 37172 61221 68743 4485 58297 81755 8914 20278 10812 7245 12396 58819 73681 59762 77740 96148 43152 95437 98021 66042 85449 32283 58442 40780 66839 95622 84380 29416 78365 19630 22848 96097 98447 35400 64245 64855 6534 46247 51336 59634 16063 87819 89892 34866 91245 18310 326 23642 26041 85120 23027 22789 33015 18180 93363 76953 71679 44306 20771 32303 38816 24886 89531 6212 34412 94327 30145 71771 3047 29545 54324 75169 2977 84793 32881 97359 65836 33198 17943 79596 15825 14861 8793 86553 31837 57169 35294 5602 30261 20971 20110 38374 61327 34148 
9822 29940 16354 21218 38646 25964 45710 17285 417 16908 27563 32197 17239 26954 34276 18945 13411 2116 28431 36479 7031 15315 45148 44365 9502 46631 1932 972 19581 24139 35465 48118 38023 43498 30749 49269 2696 33268 38246 9070 29047 23826 656 4882 22813 34778 47562 47368 17793 6217 16766 33173 38125 49263 29242 45727 41274 41845 9747 45588 9249 19549 12565 22606 47154 3120 43049 5513 7860 16588 44780 13179 33889 14377 14256 10653 37034 23262 40394 15595 38151 9463 24781 41278 36763 30926 6005 36108 2912 48594 17104 23442 44675 29721 4076 29112 42827 14610 11974 13675 19058 22081 45758 23341 43661 29886 45372 25988 24462 25786 42861 45274 41841 7635 11892 6980 21682 13670 11397 7144 17649 30945 292 46176 2165 5962 1502 39980 1028 30674 43156 24036 13654 8621 40257 19470 16756 22701 35157 39808 13520 35049 22327 24981 7244 16554 1658 14305 49174 25721 15540 20519 11806 37129 37044 29940 4606 29254 37352 21346 2536 21230 40498 5501 1755 42716 8724 1527 5358 31029 45089 25475 39120 1661 32348 43597 28253 31999 49013 14255 41092 18746 4392 13051 36847 6378 25697 47425 48654 41601 7942 40739 15404 35871 9629 27285 12425 31543 20585 41498 
200
40436 8695 44068 19610 21367 17815 24709 8171 53962 85329 45008 86380 62439 52962 90645 23751 65604 205 86883 64465 53079 4093 91330 21716 80313 90066 22310 52341 63852 10052 90878 61046 98063 55299 87785 40299 3131 77079 80048 56760 55791 90107 30438 33807 34667 50143 28289 46158 88584 31136 56691 72778 14460 65722 67620 34125 42218 48125 95176 60017 22578 12274 53102 59399 77269 92540 42008 35915 70639 24225 15590 65888 99196 60481 58162 53156 35255 82669 2004 44625 36451 72453 42463 70 10137 55641 91210 24322 42875 1774 6283 21621 84855 7249 51853 57928 36247 47624 69230 16735 74253 27794 51537 23312 26711 78611 40566 35671 71203 42944 60899 89555 39465 28472 49335 63136 64222 85051 92158 48737 33016 37992 46792 45032 31412 21575 3230 71378 78988 64149 18908 35458 15195 39930 95072 28104 71256 36277 49265 92366 29756 72503 62036 509 80406 11463 64368 22852 56726 12595 20308 49102 48680 63495 25936 44383 59375 37778 10318 24127 35991 31493 51839 66214 81672 96404 5897 57902 40899 17319 32774 52310 50684 62052 21741 47891 96273 24060 25464 94330 82355 83920 5721 85205 47337 30795 39918 55261 54532 56429 92581 96456 45694 60487 48585 28934 9353 82126 56736 16834 
29296 23135 20487 12712 5579 11451 22196 222 46071 14994 7493 37286 6445 27600 32413 22671 43526 39751 24509 27186 33278 26549 36001 18526 42896 10857 8659 40273 40198 47158 21412 12272 21551 38455 7344 32432 35750 35325 864 38795 44037 2098 49265 21380 6005 34933 30605 3838 14827 29012 669 29787 35923 21067 2341 30762 8706 16578 19440 44551 49735 9564 33628 1732 35182 1191 15489 30266 35702 29836 42571 44142 20281 15272 14853 2337 42076 2463 41052 29299 285 4415 49694 34215 36649 23578 37438 2440 25908 43981 10781 41830 24762 31228 11381 41279 19532 49457 36688 31257 17881 2124 16296 13681 42414 7823 36340 6236 36067 46575 44818 30322 36109 39231 10927 22394 42362 31196 9550 42726 47277 18788 25090 7466 13919 41643 3663 47340 8390 8636 6499 33841 30591 33049 47233 29322 26654 37519 39596 42224 20411 24041 37106 32245 16130 11242 27140 48323 35161 13059 22847 34223 33542 43613 45169 2723 3453 7914 43960 25603 4218 10606 26269 42711 34857 12791 24710 18182 14079 7668 28471 42560 43052 38829 9978 27335 1391 12373 9724 28389 21954 10047 17356 36097 42294 22407 21028 17071 1634 237 45379 14547 43921 46289 208 15651 35217 36049 14619 33890 
198
27324 13129 86713 96265 46591 57437 17205 39310 28705 12011 60004 53198 8392 41297 45576 19044 84050 25829 56418 17281 40907 80735 58489 39795 73882 6953 603 5380 58428 4455 6118 59070 60647 38420 23389 15090 18718 62307 56751 10638 53749 47668 49506 48591 23251 2882 38247 61175 59446 63189 73477 60858 82278 14251 37527 86174 32414 2202 88548 54096 858 62016 47108 74505 32360 29002 41620 91487 99332 13186 83876 97575 47899 94689 15901 88272 2703 96159 23079 48374 39667 45138 92586 73451 21643 12783 39759 77862 43229 95222 62916 26129 81368 67262 94790 97831 44965 82035 86493 80150 71530 16039 3297 61922 9316 59387 15117 3326 1177 22748 39972 12079 63117 82106 40999 50719 84746 14820 36893 48696 81599 61011 55063 24616 31476 66141 33406 6521 13974 12762 31285 89996 2033 17550 29384 11266 98819 23264 19471 11 56710 76318 76308 69034 91317 57993 91755 8076 80952 49214 94380 16157 45924 23718 7658 67386 794 88242 61896 84299 4044 52807 90000 46314 80683 53069 70231 38499 50694 89274 47060 7715 30470 20665 75365 78385 66047 83227 41021 80707 39383 25063 88987 48548 14838 99551 20969 29851 53628 46745 95590 72490 68519 84632 94709 87426 39734 5427 
5338 13940 44890 22004 66 41196 35790 24649 20859 31872 19868 29905 25004 27446 15953 11101 25863 19942 34383 21096 18918 42676 30705 41818 24457 22363 26535 33450 33249 1634 1864 34674 42328 18935 18030 4896 8959 481 7608 48656 28106 24946 21824 32342 35445 21260 28743 27572 7517 27806 8643 7216 23387 11216 49735 47686 28229 23201 38045 45065 164 43458 11767 12847 23908 10989 3214 34605 26134 34944 36917 35555 24329 35886 38567 26952 10116 10542 42309 4889 23918 49170 424 22253 26285 19646 44165 45814 40765 19723 18103 10282 13509 42025 7244 20197 25541 28507 4851 35690 29825 1600 19896 17187 4977 23083 44493 4886 19288 1946 13466 44888 43620 24832 17948 44251 28590 42911 13939 11138 28562 22717 4220 5418 483 47083 17857 46835 25780 31916 20809 10358 31336 2952 30932 20887 16895 18371 11994 24505 22207 49036 28711 46303 17920 9733 49059 31259 25069 13860 40706 42039 23626 39877 43233 28529 19611 39424 21495 29777 17320 47202 24146 48705 30948 11550 30320 48844 17951 31844 42001 42495 20211 41230 44860 28199 2628 21206 17513 48903 38750 35721 27524 25868 42454 9944 38186 20047 48093 34332 8441 35668 45947 33093 31663 9449 47569 39200 
197
55429 62428 22291 59846 11553 40406 13169 92411 21041 71962 3532 60392 70608 63793 1071 57447 79834 61941 58282 11508 55625 49322 25409 50885 7437 52697 22141 69924 75484 77798 26445 34362 21350 6879 62888 46699 5628 57980 39161 99346 50571 69685 92991 25493 14098 97955 12493 41879 8608 74652 29125 83549 14275 47018 70566 28784 30363 11611 1350 59644 41298 93060 18451 37956 43428 94640 62246 46376 33485 76770 53042 29657 68839 45188 18718 9438 85123 51143 62056 80013 97449 77628 99958 81614 64043 57886 86948 86810 21716 54667 71680 6348 74518 406 74948 13101 81041 79635 30048 69608 35345 75640 14249 15637 24627 72888 78107 10680 11623 21034 55700 22946 50061 36369 62630 86788 89491 3451 40491 65449 77984 36751 74846 14989 75136 38460 16809 20760 87297 99400 61271 72904 2267 87677 90177 45494 57069 9112 21084 40668 74063 93954 24193 82741 24385 27694 15255 32706 93253 23298 10163 62787 41434 22404 1741 74124 99027 5120 74563 45041 26634 1121 68976 67012 12315 55601 40260 18749 49394 61045 82870 61707 36511 69257 44083 12176 96419 79260 84136 36598 56510 7866 98046 52094 74107 90844 92402 52575 16776 4561 69100 50299 10972 61858 46456 72297 56547 
29804 44767 5741 15264 417 4777 16308 6930 7763 11784 12244 2591 10699 17855 31558 13428 35038 32244 23465 36844 7009 45771 37578 46248 14058 33675 5777 10661 15864 21222 40845 27928 31001 48478 19592 24386 6533 36329 18116 20070 2309 10221 23535 15351 5069 5774 17481 14714 11966 4647 30320 4459 45420 13937 23870 4378 29733 1041 45335 13048 45595 41927 6941 14743 7722 27795 45988 16737 43196 6777 31492 36504 23525 20952 4807 48794 30935 35979 1189 5210 20826 18758 80 27445 46974 15409 45190 24152 21233 39579 47985 46215 35283 37283 14983 24662 31747 42538 13269 14213 34563 31136 4385 12966 13152 33111 41947 19660 36951 38679 23735 46133 23373 6861 30717 29809 48942 23825 17088 2004 6162 21883 37892 44833 28048 7128 46001 46128 37575 13270 17263 41519 35836 21323 7864 35869 47425 21742 6698 13321 26567 47177 48244 24923 4805 9149 20172 46585 4437 36269 46122 40858 13149 45745 12664 40484 24967 27861 29351 12797 9682 40512 36311 40900 21172 45468 49179 49530 29131 35280 39764 16868 23273 13731 36118 15216 17527 2002 3994 14478 13148 38799 23008 7493 45681 28946 43349 30257 3626 35202 16415 49000 13638 12060 423 38758 30240 
198
21492 28888 52014 88836 57834 63177 42134 7376 4118 5389 73094 73426 98558 92767 56082 86269 90776 61612 70965 7047 22346 14264 27952 42506 19834 88509 85308 6764 21393 27678 14133 11569 7386 32206 37807 58939 67920 96737 86377 46330 45069 42171 26614 64097 3530 29850 86751 53443 84410 27855 51859 33029 16847 7333 48103 27301 48891 11943 60511 28262 1794 29611 12894 68863 69577 80249 8030 3087 52079 47799 89777 30439 95508 12282 35984 70713 71528 5177 75808 79085 51993 27778 6007 43307 99637 66083 75214 49853 88819 8874 80755 65006 66542 1939 54343 59522 78062 53344 75514 49259 46215 49585 91545 9481 29531 1833 78625 95959 9765 18236 38851 17085 6170 48225 52735 10198 82044 94086 35721 61397 51302 37460 66104 24481 25989 54030 5173 16227 50306 13748 33315 10733 927 46629 60776 80731 65323 75497 92898 55514 76320 22721 44014 39397 69223 59633 93315 93008 93460 18059 98479 5211 2242 42843 90824 66712 17002 11626 33264 43457 87396 41791 59815 20545 55480 37701 40058 30093 76981 48770 85610 84833 12872 18121 72432 65445 55493 17758 47072 91760 77968 41578 19330 40952 89470 12589 45572 897 22225 51326 70846 73893 72329 29119 51451 23062 96635 39629 
22745 899 34329 37653 43959 33625 16096 10770 5835 45974 47784 6602 34252 25448 48971 42805 7450 13170 39631 12588 36218 28142 3079 3589 367 18207 17277 3911 45423 28546 42220 38263 32217 27027 30216 33238 9495 48566 6050 15254 4800 31943 27951 12030 38523 26135 15664 23547 39234 24768 13818 44345 20519 35231 20413 21626 28735 8575 18396 35850 9767 43139 48661 12644 19629 10377 1953 24783 3290 34953 9472 43747 46574 538 13749 40972 1243 40053 38509 38462 13849 11318 6849 40998 12800 19069 26246 16761 20995 11701 33632 17824 46428 49387 11127 24272 25451 47096 46214 32566 14077 30985 18180 36530 43314 16198 48374 44413 49785 40922 27612 5753 14861 32412 22605 40754 4188 38904 24138 3718 5537 30302 21041 9450 49287 6175 43540 38929 7363 44355 44110 20655 11960 1419 8480 40424 6873 38850 37460 12644 19428 6977 15151 49833 5460 22448 35013 33543 35837 5373 3512 31434 26591 25001 47957 29947 27788 14481 39773 34841 48233 7601 28197 21747 6021 32759 3256 48829 2527 9463 27319 45234 3357 33990 23653 16855 31631 18972 40024 2757 1987 46655 35163 516 37379 213 6901 26511 43047 27007 42029 32605 21275 47124 20287 40609 25667 10108 
