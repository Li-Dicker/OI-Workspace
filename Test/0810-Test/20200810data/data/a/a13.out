Case #1: 14861686
Case #2: 14200170
Case #3: 13820236
Case #4: 14107627
Case #5: 13967325
Case #6: 14329625
Case #7: 13943561
Case #8: 14171276
Case #9: 13985227
Case #10: 13505648
